cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -279,9 +279,24 @@ message LineBasedExtent {
 
 enum SurveyCoverageSource {
     // Used as the default when a specific source isn't requested by the user.
-    UNSPECIFIED = 0;
+    UNSPECIFIED_SURVEY_COVERAGE_SOURCE = 0;
```

Comment:
Why the rename?

Reply:
Protobuf is not happy with repeated enum field names 😞

Reply:
😔